Packages/vcs/Lib/vtk_ui/behaviors/behavior.py

```diff
@@ -0,0 +1,51 @@
+class Behavior(object):
+    """
+    Base class; requires subclasses to have an interactor property
+    """
+
+    def __init__(self):
+
+        self.events = {}
+        self.listeners = {}
+        super(Behavior, self).__init__()
+
+    def event_position(self, normalized=True):
+        point = self.interactor.GetEventPosition()
+        if normalized:
+            w, h = self.interactor.GetRenderWindow().GetSize()
+            x, y = point
+            x = x / float(w)
```

Comment:
may be check if w and h are not zero. If the window is not initialized or in a bad state you may get that.